Source/JavaScriptCore/yarr/YarrCanonicalizeUCS2.cpp

```diff
@@ -81,7 +81,7 @@ const char32_t* const ucs2CharacterSetInfo[UCS2_CANONICALIZATION_SETS] = {
     ucs2CharacterSet22,
 };
 
-const size_t UCS2_CANONICALIZATION_RANGES = 448;
+const size_t UCS2_CANONICALIZATION_RANGES = 460;
```

Comment:
style-checker warn the following error here but this source is generated code.

```
ERROR: Source/JavaScriptCore/yarr/YarrCanonicalizeUCS2.cpp:84:  UCS2_CANONICALIZATION_RANGES is incorrectly named. Don't use underscores in your identifier names.  [readability/naming/underscores] [4]
```

Reply:
Typically when we run into something like this, we update the style checker script so it knows about exceptions, or tweak the script to change the generated code.

Reply:
I see. I updated the codegen to match the style checker.